booking-deeplink-specification.md

```diff
@@ -0,0 +1,24 @@
+# Indroduction
+
+This specification provides precise guidelines for a carpooling operator and a MaaS platform to implement a "booking" flow for passengers, with information to complete the booking shared via deeplinking between the two.
+
+## Requirements Notation and Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "PROVIDERTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+## Functional overview
+
+The booking flow with a deep link starts when a passenger clicks on a journey search result in the MaaS app. This search result MUST open the deep link contained in the [DeepLink](https://github.com/klaxit/open-carpooling-service-specification/blob/c34cbfdea9991bfd3453d0d0822066411dcfa219/carpool-api-en.yml#L268) parameter returned by the [CarpoolJourney](https://github.com/klaxit/open-carpooling-service-specification/blob/c34cbfdea9991bfd3453d0d0822066411dcfa219/carpool-api-en.yml#L140) object in the [carpoolJourneys](https://github.com/klaxit/open-carpooling-service-specification/blob/master/carpool-api-en.yml#L10) response. This parameter MUST be provided by the carpool operator responsible for their implementation of the search API public specification. It contains the full URL and its corresponding parameters needed for the mobile device to redirect the passenger from the app displaying the search results to the operator's app.
```

Comment:
@riclage please see [my comment](https://github.com/fabmob/standard-covoiturage/pull/2/files#r832663218) describing why the `webUrl` should be the preferred way from my point of view.

If everyone agree that only the `webUrl` should be kept, it will be much simplier. If not, both methods would have be supported and described here...

Reply:
> webUrl should be the preferred way from my point of view

OK for me as long as we update its description on the other spec to explicitly state it contains the deeplink to start the booking flow.

Reply:
> OK for me as long as we update its description on the other spec to explicitly state it contains the deeplink to start the booking flow.

I tried that but I not sure it will fit, please tell me if you see otherwise:
https://github.com/fabmob/standard-covoiturage/pull/8/files#r833143774

Reply:
Yes, OK to leave it like that. I replied there.